oat-sa/tao-test-runner-qti-fe
# [ACTP-300] Main Layout - BottomBar - Fix the tools labelling in the bottom bar

Related to : https://oat-sa.atlassian.net/browse/ACTP-300
 
- Fix the tools labelling in the bottom bar
- Refactor bottom bar to have separated landmarks for toolbox and navigation
 
#### How to test
 
- prepare an instance of TAO with taoAct extension
- prepare a delivery with review panel enabled
- execute the delivery as a test taker
- make sure that screen reader properly announce toolbox button
- make sure that toolbox and navigation have separated landmarks

#### Dependencies 

Requires :
 - [ ] https://github.com/oat-sa/tao-test-runner-qti-fe/pull/141
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1331
